CHANGES.md

```diff
@@ -239,6 +239,8 @@
 - Fix `SearchBoxAndResults` Trans values
 - Fix `generateCatalogIndex` for nested references
 - Fix `SearchBox` handling of `searchWithDebounce` when `debounceDuration` prop changes. It now fushes instead of cancels.
+- Move map credits to map column so it don't get hidden by chart panel
+- [The next improvement]
```

Comment:
Hey @steve9164 and @staffordsmith83 

Before this is merged in can you please update the `CHANGES` file here. Thanks!